packages/documentation/public/_headers

```diff
@@ -3,3 +3,21 @@
 
 /assets/versions.json
   Access-Control-Allow-Origin: *
+
+/assets/*.css
+  Access-Control-Allow-Origin: https://cdpn.io
+
+/post-icons/*.svg
+  Access-Control-Allow-Origin: https://cdpn.io
+
+/post-icons/*.png
+  Access-Control-Allow-Origin: https://cdpn.io
+
+/post-icons/*.jpg
+  Access-Control-Allow-Origin: https://cdpn.io
+
+/post-icons/*.jpeg
+  Access-Control-Allow-Origin: https://cdpn.io
+
+/post-icons/*.gif
+  Access-Control-Allow-Origin: https://cdpn.io
```

Comment:
Nope. There are for sure no png, jpg or gifs in the `post-icons` folder.
However, we have some (svg, png, jpg, gif) in the public folders `/assets` and some in `/images`.